utils/common/types/provider.go

```diff
@@ -0,0 +1,5 @@
+package types
+
+const (
+	EMPTY_PROVIDER = "empty_provider"
```

Comment:
redundant const

Reply:
Thanks for your reply! But cycle imports are not allowed. I think it will require more changes feel free to commit to my branch